embark-skills.lua

```diff
@@ -42,13 +42,13 @@ function adjust(dwarves, callback)
 end
 
 local scr = dfhack.gui.getCurViewscreen() --as:df.viewscreen_setupdwarfgamest
-if dfhack.gui.getCurFocus() ~= 'setupdwarfgame' or scr.show_play_now == 1 then
+if not df.viewscreen_setupdwarfgamest:is_instance(scr) then
```

Comment:
> FYI: `dfhack.gui.matchFocusString()` is the most direct equivalent (getCurFocus returns a list now because multiple contexts can be visible at once in fortress mode), but I personally prefer your method because it is a more direct check.

Ahh gotcha thanks for letting me know.